modules/crowdx/src/lib/index.ts

```diff
@@ -30,3 +30,13 @@ export interface IObservable<ValueT = any> {
 export interface IObserver<ValueT = any> {
   onUpdate: IObserverHandler<ValueT>
 }
+
+export enum Kind {
+  Observable,
```

Comment:
When a value of this kind will be accessed it will be 0, 1 or 2.  Maybe give the keys a value like Observable = 'observable', Computed = 'computed', Dervived = 'derived'. This way it will be much easier to understand what the value actually means.

Reply:
Kinds have been removed all together